tests/system/python/conftest.py

```diff
@@ -30,18 +31,21 @@
 
 @pytest.fixture
 def clean_setup_fledge_packages(package_build_version):
-    assert os.environ.get('FLEDGE_ROOT') is not None
```

Comment:
why this has been removed?

Reply:
This is removed because this def expects `FLEDGE_ROOT` to be the path of the cloned fledge directory because it tries to run the script from that path. But in the case of packages `FLEDGE_ROOT` should be `/usr/local/fledge`. So added `PROJECT_ROOT` to run the scripts with reference to the `PROJECT_ROOT` path.